packages/shared/src/types/props/header.ts

```diff
@@ -23,6 +24,7 @@ export interface HeaderProps {
     countryCode?: string;
     submitComponentEvent?: (componentEvent: OphanComponentEvent) => void;
     numArticles?: number;
+    prices?: Prices;
```

Comment:
we don't actually support any templates for the header currently, so I think it's best to remove this

Reply:
Removed prices templates functionality from Headers